lib/combineReducers.spec.lua

```diff
@@ -0,0 +1,22 @@
+return function()
+	local combineReducers = require(script.Parent.combineReducers)
+
+	it("should invoke each sub-reducer for every action", function()
+		local aCount = 0
+		local bCount = 0
+
+		local reducer = combineReducers({
+			a = function(state, action)
+				aCount = aCount + 1
+			end,
+			b = function(state, action)
+				bCount = bCount + 1
+			end,
```

Comment:
For pedantry's sake, can we actually return a new state value for each of these to test that values are assigned to the correct key? Something simple, like making `a` return `state + 1` and `b` returning `state + 2` would do.

Reply:
Should be good now!